### PR TITLE
LPS-156173 [App Pages] Validate captcha only when is present captcha fragment

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-156173

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/AddInfoItemStrutsAction.java [hunk 1]
```diff
@@ -158,9 +180,154 @@ protected void activate() {
...
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		LayoutStructure layoutStructure =
+			LayoutStructureUtil.getLayoutStructure(
+				ParamUtil.getLong(
+					httpServletRequest, "plid", themeDisplay.getPlid()),
+				ParamUtil.getLong(
+					httpServletRequest, "segmentsExperienceId",
+					themeDisplay.getPlid()));
```
Comment (ealonso) [inline modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/AddInfoItemStrutsAction.java:298]: themeDisplay.getPlid, seems to be wrong

Comment (ealonso) [reply modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/AddInfoItemStrutsAction.java:298]: cc: @lfbesada

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/AddInfoItemStrutsAction.java (hunk 1 above)
Comment (lfbesada) [reply modules/apps/info/info-impl/src/main/java/com/liferay/info/internal/request/struts/AddInfoItemStrutsAction.java:298]: that's true! thanks for catching it!

**Comment by ealonso** [comment]
see: https://github.com/lfbesada/liferay-portal/pull/85

**Pull request closed without merging**